pkg/gridtypes/workload.go

```diff
@@ -194,6 +194,10 @@ func (w *Workload) Challenge(i io.Writer) error {
 		return err
 	}
 
+	if _, err := fmt.Fprintf(i, "%s", w.Name); err != nil {
```

Comment:
I checked on main branch, but it seems to be new. Isn't that a breaking change? won't that need an update in clients?

Reply:
yes yes, i already talked to Hanafy and explained to him and he said it's okay and he will prepare the fix from his side

Reply:
okay CC @ashraffouda / @rawdaGastan please check if you need to update anything as well